demosplan/DemosPlanCoreBundle/Security/Authentication/Authenticator/OzgKeycloakAuthenticator.php

```diff
@@ -35,7 +35,7 @@ class OzgKeycloakAuthenticator extends OAuth2Authenticator implements Authentica
     public function __construct(
         private readonly ClientRegistry $clientRegistry,
```

Comment:
The "p" in AuthenticationEntrypointInterface should be uppercase to match declaration.

Reply:
f67bf45